python/ray/tests/test_autoscaler_yaml.py

```diff
@@ -175,7 +175,7 @@ def testValidateDefaultConfigAWSMultiNodeTypes(self):
             "CPU": 4,
             "memory": 12025908428,
             "neuron_cores": 2,
-            "accelerator_type:aws-neuron-core": 2,
+            "accelerator_type:aws-neuron-core": 1,
```

Comment:
why is it changed?

Reply:
It's a bug from the previous PR. Total quantity of the special accelerator_type resource should only be 1.